### DISCUSSION TITLE
Adding Custom Error Pages

### DISCUSSION THREAD

**Discussion by harblaith7** [nuxt/nuxt#18111] [Questions]
I am very confused about how to add custom error pages rather than using the default. I tried reading nuxt/nuxt#17750 but it was a little confusing. Can someone show me how to do it by example?

**Comment by maxdzin** [comment]
@harblaith7 For the Nuxt v3 app you can create `pages/404.vue`.
But there is the problem that it returns the response with status code `200` rather than `404` as expected.
The related issue is described here: https://github.com/nuxt/framework/issues/2278
